src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -1347,6 +1354,7 @@ internal IEnumerable<PackageManagerSearchElementViewModel> GetAllPackages()
                 list.Reverse();
             }
 
+            // TODO - should we be unsubsribing somewhere?
```

Comment:
Do we need to review this?

Reply:
Nice! OK, I wasn't sure it works like that.

Reply:
I just spotted that there are a few private properties that are now public with a TODO comment for 3.0 release. I am not sure who made the comments, or the properties, but should I take care of these in this PR?